test/integration/test-helper.coffee

```diff
@@ -0,0 +1,39 @@
+_ = require 'underscore'
+Promise = require 'bluebird'
+
+module.exports.unpublishProduct = (logger, client, product) ->
```

Comment:
@hisabimbola this doesn't work:
`exports = function nope () {}`
this does:
`module.exports = function yes () {}`